diagnostic.md

```diff
@@ -36,18 +36,21 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+It's an relative path because its an instructed path on your local computer or
```

Comment:
this is absolute, and easy way to tell is that it give the exact roadmap to the location without and other the `..`